wasmcloud-chat/actors/nats-channel/Makefile

```diff
@@ -0,0 +1,34 @@
+COLOR ?= always # Valid COLOR options: {always, auto, never}
+CARGO = cargo --color $(COLOR)
+TARGET = target/wasm32-unknown-unknown
+DEBUG = $(TARGET)/debug
+RELEASE = $(TARGET)/release
+KEYDIR ?= .keys
+VERSION = $(shell cargo metadata --no-deps --format-version 1 | jq -r '.packages[].version')
+
+.PHONY: all build check clean doc test update
+
+all: build
+
+build:
+	@$(CARGO) build	
+	wash claims sign $(DEBUG)/nats_channel.wasm -c wasmcloud:httpserver --name "nats_channel" --ver $(VERSION) --rev 0
+
+check:
+	@$(CARGO) check
+
+clean:
+	@$(CARGO) clean
+
+doc:
+	@$(CARGO) doc
+
+test: build
+	@$(CARGO) test
+
+update:
+	@$(CARGO) update
+
+release:
+	@$(CARGO) build --release	
+	wash claims sign $(RELEASE)/nats_channel.wasm -c wasmcloud:httpserver --name "nats_channel" --ver $(VERSION) --rev 0
```

Comment:
Same comment as messages, the capability attestations will likely just be `wasmcloud:messaging` I'd imagine

Reply:
nats channel may only need that claim, but it might actually need 2 instances of linked providers.. ;) Will have to see how that design turns out.